CHANGELOG.md

```diff
@@ -2,6 +2,77 @@
 
 IMPROVEMENTS:
 * CRDs: update the CRD versions from v1beta1 to v1. [[GH-464](https://github.com/hashicorp/consul-k8s/pull/464)]
+* Connect: the connect-inject init container has been split into two init containers. [[GH-441](https://github.com/hashicorp/consul-k8s/pull/441)]
+
+* Connect: A new internal command `consul-k8s connect-init` has been added. It replaces the existing init container logic for ACL login and Envoy bootstrapping and introduces a polling wait for service registration, see `Endpoints Controller` for more information. [[GH-446](https://github.com/hashicorp/consul-k8s/pull/446)], [[GH-452](https://github.com/hashicorp/consul-k8s/pull/452)], [[GH-459](https://github.com/hashicorp/consul-k8s/pull/459)]
+  
+* Connect: A new controller `Endpoints Controller` has been added which is repsonsible for managing service endpoints and service registration. When a Kubernetes service referencing a connect-injected deployment is deployed the endpoints controller will be responsible for managing the lifecycle of the connect-injected deployment. [[GH-455](https://github.com/hashicorp/consul-k8s/pull/455)], [[GH-467](https://github.com/hashicorp/consul-k8s/pull/467)], [[GH-470](https://github.com/hashicorp/consul-k8s/pull/470)], [[GH-475](https://github.com/hashicorp/consul-k8s/pull/475)]
+  - This includes:
+      - service registration and deregistration, formerly managed by the `init-container`.
+      - monitoring health checks, formerly managed by `healthchecks controller`.
+      - re-registering services in the events of consul agent failures, formerly managed by `consul sidecar` and the `cleanup controller`.
+
+  - The endpoints controller replaces the health checks controller while preserving existing functionality. [[GH-472](https://github.com/hashicorp/consul-k8s/pull/472)]
+
+  - The endpoints controller replaces the cleanup controller while preserving existing functionality. [[GH-476](https://github.com/hashicorp/consul-k8s/pull/476)], [[GH-454](https://github.com/hashicorp/consul-k8s/pull/454)]
+
+  - Merged metrics configuration support is now partially managed by the endpoints controller.  [[GH-469](https://github.com/hashicorp/consul-k8s/pull/469)]
+
+* Connect: Leader election support for connect webhook and controller deployment. [[GH-479](https://github.com/hashicorp/consul-k8s/pull/479)]
+
+* Connect: Connect webhook no longer generates its own certificates and relies on them being provided as files on the disk. [[GH-454](https://github.com/hashicorp/consul-k8s/pull/454)]] 
+
+* Connect: Connect pods and their Envoy sidecars no longer have a preStop hook as service deregistration is managed by the endpoints controller.  [[GH-467](https://github.com/hashicorp/consul-k8s/pull/467)]
+
+* Connect: connect-inject webhook has been refactored to use controller-runtime manager. [[GH-454](https://github.com/hashicorp/consul-k8s/pull/454)]
```

Comment:
Not sure we need this because the changelog is for user-facing functionality changes.